API/SearchService/src/util.py

```diff
@@ -0,0 +1,73 @@
+from enum import Enum
+import azure.functions as func
+
+from .exception import ParameterException
+
+class SearchType(int, Enum):
+    All = 1
+    Scenario = 2
+    Command = 3
+
+    def get_search_fields(self):
+        if self == self.All:
+            return ["scenario", "description", "commandSet/command"]
+        elif self == self.Scenario:
+            return ["scenario", "description"]
+        elif self == self.Command:
+            return ["commandSet/command"]
+
+
+class RequiredParameter:
+    pass
```

Comment:
It is not intuitive to use it to indicate whether it is a required parameter, and it will logically become a required parameter if there is no default value